docs/tech-preview.rst

```diff
@@ -0,0 +1,9 @@
+Tech previews
+=============
+
+The following features are currently being released as part of a tech preview
+
+* New endpoint “pulp/api/v3/remotes/python/python/from_bandersnatch/” that allows for Python remote creation from a
+  Bandersnatch config file.
+* PyPi’s json API at content endpoint ‘/pypi/{package-name}/json’. Allows for basic Pulp-to-Pulp syncing.
+* Fully mirror Python repositories like PyPi
```

Comment:
I'm not sure that the "fully mirror" use case should be considered tech preview, it may still have bugs, but the actual user workflow isn't likely to change at all, which is the main concern